index.html

```diff
@@ -7,93 +7,211 @@
+                            </ul>
+                        </div>
+
+                        <div class="frame">
+                            <h4>Frameworks</h4>
+                            <ul>
+                                <li>Flask</li>
+                                <li>Ruby on Rails</li>
+                                <li>SQlite</li>
+                                <li>Bootstrap</li>
+                            </ul>
+                        </div>
+
+                        <div class="frame">
+                            <h4>Skills</h4>
+                            <ul>
+                                <li>Codekit</li>
+                                <li>Github</li>
+                                <li>Gitlab</li>
+                                <li>Codepen</li>
+                                <li>Terminal</li>
+                            </ul>
+                        </div>
                     </div>
-                </div>
 
-                <hr class="divider">
+                    <hr class="divider">
 
-                <div class="spprtItems flex column">
-                    <div class="socialMedia flex">
-                        <a class="github flex" role="button" href=""><img src="icons&imgs/Icon-GitHub.png"
-                                alt="github"></a>
+                    <div class="spprtItems flex column">
+                        <div class="socialMedia flex">
+                            <a class="github flex" role="button" href=""><img class="comunic"
+                                    src="icons&imgs/Icon-GitHub.png" alt="github"></a>
 
-                        <a class="linkedin flex" role="button" href=""><img src="icons&imgs/Icon-Linkedin.png"
-                                alt="linkedin"></a>
+                            <a class="linkedin flex" role="button" href=""><img class="comunic"
+                                    src="icons&imgs/Icon-Linkedin.png" alt="linkedin"></a>
 
-                        <a class="angelist flex" role="button" href=""><img src="icons&imgs/angelist.png"
-                                alt="angelist"></a>
+                            <a class="angelist flex" role="button" href=""><img class="comunic"
+                                    src="icons&imgs/angelist.png" alt="angelist"></a>
 
-                        <a class="twitter flex" role="button" href=""><img src="icons&imgs/twitter.png"
-                                alt="twitter"></a>
+                            <a class="twitter flex" role="button" href=""><img class="comunic"
+                                    src="icons&imgs/twitter.png" alt="twitter"></a>
 
-                        <a class="medium flex" role="button" href=""><img src="icons&imgs/Vector.png" alt="medium"></a>
+                            <a class="medium flex" role="button" href=""><img class="comunic"
+                                    src="icons&imgs/Vector.png" alt="medium"></a>
+                        </div>
```

Comment:
- [ ] The social media icons should be in an unordered list `ul` and each link inside an `li`, please change that to what I mentioned. 😁